File: backend/team6/src/test/java/programmers/team6/support/JwtMemberInfoMother.java
package programmers.team6.support;

import programmers.team6.domain.auth.dto.JwtMemberInfo;
import programmers.team6.domain.member.enums.Role;

public class JwtMemberInfoMother {

    public static JwtMemberInfo defaultUser() {
        return new JwtMemberInfo(1L, "member1", Role.USER);
    }

    public static JwtMemberInfo admin() {
        return new JwtMemberInfo(2L, "admin", Role.ADMIN);
    }

}

File: backend/team6/src/test/java/programmers/team6/support/JwtMemberInfoMother.kt
package programmers.team6.support

import programmers.team6.domain.auth.dto.JwtMemberInfo
import programmers.team6.domain.member.enums.Role

object JwtMemberInfoMother {
    @JvmStatic
    fun defaultUser(): JwtMemberInfo {
        return JwtMemberInfo(1L, "member1", Role.USER)
    }

    fun admin(): JwtMemberInfo {
        return JwtMemberInfo(2L, "admin", Role.ADMIN)
    }
}
